In a cross compile environment adding paths like /usr are not correct.
With this patch we make it so that its relative to sysroot if the compiler
defines one.

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
Index: a/src/Makefile.am
===================================================================
--- a/src/Makefile.am	2013-02-08 22:35:08.000000000 -0800
+++ b/src/Makefile.am	2013-10-13 01:31:38.812869080 -0700
@@ -33,5 +33,5 @@
 	video_encode.c video_encode.h \
 	subtitles.c subtitles.h 
 
-coriander_LDADD = -L/usr/X11R6/lib/ @PACKAGE_LIBS@ $(INTLLIBS) $(LIBTIFF_LIBS) $(FTPLIB_LIBS) $(SDLLIB_LIBS) $(LIBDC_LIBS) $(LIBRAW_LIBS) $(XV_LIBS) $(COR_LFS_LDFLAGS) -lgthread-2.0 $(FFMPEG_LIBS) $(X11_LIBS)
+coriander_LDADD = -L=/usr/X11R6/lib/ @PACKAGE_LIBS@ $(INTLLIBS) $(LIBTIFF_LIBS) $(FTPLIB_LIBS) $(SDLLIB_LIBS) $(LIBDC_LIBS) $(LIBRAW_LIBS) $(XV_LIBS) $(COR_LFS_LDFLAGS) -lgthread-2.0 $(FFMPEG_LIBS) $(X11_LIBS)
 
